source/MaterialXContrib/MaterialXNode/MaterialXNode.h

```diff
@@ -61,6 +84,9 @@ class MaterialXNode : public MPxNode
 
     MString _documentFilePath, _elementPath;
 
+    MString _envRadianceFileName = "san_giuseppe_bridge.hdr";
```

Comment:
@bernardkwok actually, with internal attributes providing the defaults is even simpler. If the value is not set by Maya then the values set in the constructor become the initial values of the attributes. I've verified that the default values are correctly populated on older assets without these attributes saved in the scene.

Reply:
The defaults themselves don't have to be hardcoded either - we could populate them from an optionVar. Worth doing at this point?